camkes/templates/component.common.c

```diff
@@ -42,6 +42,14 @@
 
 /*? macros.show_includes(me.type.includes) ?*/
 
+/*- if 'pcpus' in configuration[me.name].keys() -*/
+    /*- for c in configuration[me.name].get('pcpus') -*/
+#if(/*? c ?*/  >= CONFIG_MAX_NUM_NODES)
+#error "Invalid CPU value of /*? c ?*/ in PCPU list"
```

Comment:
let's make this message a bit more verbose:
```suggestion
#error "Invalid CPU number /*? c ?*/ in PCPU list of /* me.name */"
```

Reply:
Done